ecosystem/web-wallet/public/index.html

```diff
@@ -7,7 +7,7 @@
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <title>Aptos Wallet</title>
   </head>
-  <body>
+  <body style="margin:0; width:375px; height:600px;">
```

Comment:
Does the extension automatically put everything in a 375px by 600px box or do you have to specify how much space the wallet UI will take?

Reply:
There is a max for extensions. I just picked some roundish numbers, we can change this if we need some more room until we get some better designs.

The max is 800x600

Reply:
It can also change for each screen. Again, happy to have it be dynamic while develop if that would be useful for you.